oslo/_utils/__init__.py

```diff
@@ -0,0 +1,9 @@
+class Singleton(type):
```

Comment:
@hmy831004 I removed this file, but you restored. please remove this file.

Reply:
Ok, Sorry